scripts/mods/mod_reforged/dynamic_spawns/units/bandit_units.nut

```diff
@@ -7,6 +7,14 @@ figure_bandit_04 - Bandit Leader
 
 // We add our new bandit units
 local units = [
+	{
+		ID = "Bandit.Thug",
+		CostAdd = 2
```

Comment:
Where does this `CostAdd` come from? I don't find it anywhere in the codebase of DynamicSpawns or in your commit.